examples/basic-recipe.rs

```diff
@@ -30,8 +30,9 @@ fn main() {
 
     // set up Soup via recipe
     let mut builder = ControllerBuilder::default();
-    builder.log_with(distributary::logger_pls());
+    //builder.log_with(distributary::logger_pls());
     builder.set_worker_threads(2);
+    builder.disable_partial();
```

Comment:
We should probably not change the provided example code this way :) Maybe make a test that executes this, and leave the example code the way it was?